src/feedback/data/sagas.js

```diff
@@ -15,7 +15,13 @@ export function* handleErrors(e, clearExistingMessages) {
   if (e.errors !== undefined) {
     for (let i = 0; i < e.errors.length; i++) { // eslint-disable-line no-plusplus
       const error = e.errors[i];
-      yield put(addMessage(error.code, error.userMessage, error.data, error.messageType));
+      if (error.code === 'sku-error-message') {
+        yield put(addMessage(error.code, error.userMessage, {}, MESSAGE_TYPES.ERROR));
+      } else if (error.data === undefined && error.messageType === null) {
+        yield put(addMessage('transaction-declined-message', error.userMessage, {}, MESSAGE_TYPES.ERROR));
+      } else {
```

Comment:
I was wondering if we had something like `message_utils` in the backend! Thanks for the link! It would've been helpful before this was done lol. 

I will add this to a ticket so we keep error messages consistent.